scripts/e2e-test.sh

```diff
@@ -20,6 +20,13 @@ make install-and-wait-cert-manager
 log "Building latest version of the operator and installing it via Helm..."
 make helm-install-local
 
+for i in {1..5}; do
```

Comment:
I'm going to plus one this but seeing a sleep 15 in here makes me concerned that we're creating something flaky for the future.

Reply:
this is a leftover, I couldn't make it work in GHA Ubuntu, and I disabled it for now, so the script is not used anywhere actually